src/components/Editor/index.js

```diff
@@ -72,6 +72,10 @@ type EditorState = {
   selectedExpression: ?Object
 };
 
+function hasClass(el, classnames) {
+  return classnames.some(className => el.classList.contains(className));
```

Comment:
nice. I think i'll switch to just doing. is it a child of cm-line lol. I kinda want all tokens